metrics.go

```diff
@@ -65,24 +65,48 @@ var (
 		Buckets: durationPerBlockPerPeerHistogram,
 	})
 
+	fetchDurationPerCarPerPeerSuccessMetric = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name:    prometheus.BuildFQName("ipfs", "caboose", "fetch_duration_car_peer_success"),
+		Help:    "Latency observed during successful caboose car fetches from a single peer",
+		Buckets: durationPerBlockPerPeerHistogram,
```

Comment:
:thought_balloon:  we will need separate histogram buckets for CAR requests. Fetching bigger dags will take longer than 19s or 60s, may  run for arbitrary lengths of time.

I don't know what is the sensible max, we may start with 30min and look at the distribution once the data starts flowing:

```go
durationPerCarHistogram = prometheus.ExponentialBucketsRange(50, 1800000, 10)
```

Given the ambiguous range, I feel we can reuse the same bucket definition for both `duration_car_success` and `duration_car_peer_success`